test/cli/dev.js

```diff
@@ -481,3 +481,77 @@ test('`fusion dev` app with split translations integration (cached)', async t =>
 
   t.end();
 });
+
+test('`fusion dev` proxy gracefully recovers from cached SSR errors', async t => {
+  const dir = path.resolve(__dirname, '../fixtures/server-startup-control');
+
+  async function waitForCompileToStart() {
+    // Should figure out a better heuristic for this.
```

Comment:
Would this strategy work for your use case?

https://github.com/fusionjs/fusion-cli/blob/bd180b7ec79ed45cd475f4159d28421d668a2b01/test/cli/dev.js#L404-L417